src/index.d.ts

```diff
@@ -0,0 +1,96 @@
+
+declare const dayjs: dayjs
+export default dayjs
```

Comment:
I've change this to default export in v1.8.1 and caused a huge problem #475

Reply:
I changed it in the PR to be the same as it is today (`export = dayjs`).
Should work with both `import dayjs from 'dayjs` and `import * as dayjs from 'dayjs'`.